codalab/lib/bundle_cli.py

```diff
@@ -2027,7 +2027,16 @@ def mimic(self, args):
         Use args.bundles to generate a call to bundle_util.mimic_bundles()
         """
         client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec)
-        bundle_uuids = self.resolve_bundle_specs(client, worksheet_uuid, args.bundles)
+        try:
+            bundle_uuids = self.resolve_bundle_specs(client, worksheet_uuid, args.bundles)
```

Comment:
`resolve_bundle_specs` is kind of a confusing name...it calls `resolve_target` which calls `resolve_bundle_uuids`...can we rename / document this more clearly?